Tests_v3/Dummy data/MessagePayload.swift

```diff
@@ -9,22 +9,24 @@ extension MessagePayload {
     /// Creates a dummy `MessagePayload` with the given `messageId` and `userId` of the author.
     static func dummy<T: ExtraDataTypes>(
         messageId: MessageId,
+        parentId: MessageId? = nil,
+        showReplyInChannel: Bool = false,
         authorUserId: UserId,
         text: String = .unique,
         extraData: T.Message = .defaultValue
     ) -> MessagePayload<T> where T.User == NameAndImageExtraData {
         .init(
             id: messageId,
-            type: .regular,
+            type: parentId == nil ? .regular : .reply,
```

Comment:
What do you think about extending the `reply` case with the parent `messageId`? It would remove a possible edge case where the type is reply but a parent id is missing.